R/apply_tfrmt.R

```diff
@@ -72,41 +56,119 @@ apply_tfrmt <- function(.data, tfrmt, mock = FALSE){
       tfrmt$label,
       fail_desc = "Unable to apply row group structure"
     ) %>%
+    tentative_process(
+      apply_page_plan,
+      tfrmt$page_plan,
+      tfrmt$group,
+      tfrmt$label,
+      tfrmt$row_grp_plan$label_loc) #$location)
+
+  # if big_n is to be by page, check that the big N's match the # of tables,
+  # with the right groups
+  check_big_n_page(big_n_df, tbl_dat_wide_processed, tfrmt)
+
+  # Results of col plan
+  col_plan_vars <- tentative_process(
+    names(tbl_dat_wide),
+    create_col_order,
+    cp = tfrmt$col_plan,
+    columns = tfrmt$column,
+    fail_desc = "Unable to create dataset subset vars"
+  ) %>%
+    as_vars()
+
+  # apply remaining processing to single table or map over list of tables (paginated)
+  apply_tfrmt_subtable_mapper(tfrmt, tbl_dat_wide_processed, col_plan_vars, big_n_df)
+
+}
+
+
+#' Process the remainder of table or list of tables
+#'
+#' @param tfrmt tfrmt object
+#' @param .data processed wide tbl or list of processed wide tbls
+#' @param col_plan_vars col plan converted to vars() to be applied
+#' @param big_n_df tbl of big Ns to apply (if 1 set of overall Ns) or list of tbl of bigNs (if grouping by page)
+#' to apply
+#'
+#' @importFrom purrr map map2
+#' @noRd
+apply_tfrmt_subtable_mapper <- function(tfrmt, .data, col_plan_vars, big_n_df){
+
+  if (inherits(.data, "list") && length(.data)>1){
+    if (inherits(.data, "list") && inherits(big_n_df, "list") && length(big_n_df)>1){
```

Comment:
Can you add a comment this is for if there are multiple different big_n headings